css/style.css

```diff
@@ -30,146 +32,110 @@ a.ui-button:active,
 }
 .projectbox{
 	float:left;
-	height: 200px; 
-	width: 120px; 
+	height: 200px;
+	width: 120px;
 	border: 1px solid black;
 	padding-right: 5%;
 	margin: 3px;
 	padding-left: 5%;
 }
 .projectimage{
-	width:80%; 
+	width:80%;
 	padding: 10%;
 	max-height: 130px;
 }
 
 .ui-helper-reset {
-    font-size:10pt;
+	font-size:10pt;
 }
 
 .ui-accordion .ui-accordion-content {
-    padding: 1em 1.2em;
+	padding: 1em 1.2em;
 }
 
 .projectUsername {
-  color: black;
-}
-
-.home {
-	float:right;
-	height: 20px;
-	width:20px;
-	padding-right: 1%;
-	padding-top: 1%;
-}
-
-.panel-group {
-  margin: 10px;
-  background-color: #8fc202;
-  border-top:6px solid #8fc202;
-  border-bottom:6px solid #8fc202;
-  border-radius: 6px;
-}
-
-.panel-default>.panel-heading {
-  background-color: #8fc202;
-  border-color: #8fc202;
+	color: black;
 }
 
-.plus {
-  float:right;
-  text-shadow: 0px 0px 1px #404040;
-}
-
-.panel-default {
-  border-color: #8fc202;
-}
-
-.panel-title {
-  color: #ffffff;
-}
-
-.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
-  border-width: 0px;
-}
-
-a:hover {
-  color: #8fc202;
-  font-size: 12px;
-  cursor: pointer;
-}
+/* Inline blocks properties */
 
-a {
-  color: #8fc202;
-  font-size: 12px;
+.control {
+	background-color: #b18e35;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.panel-text {
```

Comment:
The panel-text class is also referenced in blocks/blocks.html and howtos/howtos.html. I'm not sure that we want to remove these two rules from this CSS file.

Reply:
For those tutorials in main directory, I believe all referenced via index.html, I renamed css/style.css to css/mainstyle.css and referenced that from index.html. I believe blocks.html and howto.html are loaded through index, so should use mainstyle.css and get those classes that way.

Reply:
I doubled checked and believe that you are correct.